src/databricks_ai_bridge/genie.py

```diff
@@ -62,13 +64,16 @@ def _parse_query_result(resp, truncate_results) -> Union[str, pd.DataFrame]:
         rows.append(row)
 
     dataframe = pd.DataFrame(rows, columns=header)
+    # Default to dataframe
+    query_result = dataframe
 
```

Comment:
[nit] can we just do:

if return_pandas:
    return dataframe


This is just to have better code instead of checking if a boolean is False